format/Message.fbs

```diff
@@ -64,8 +64,11 @@ table Date {
 table Time {
 }
 
+enum TimeUnit: short { MILLISECOND, MICROSECOND }
```

Comment:
I added millis and micros because those are the ones that Parquet supports.
For now nanos and second metadata would be lost in saving to Parquet and stored as int?